maintainers/maintainer-list.nix

```diff
@@ -18467,6 +18467,13 @@
     githubId = 246631;
     keys = [ { fingerprint = "3E46 7EF1 54AA A1D0 C7DF  A694 E45C B17F 1940 CA52"; } ];
   };
+  prestonhager = {
```

Comment:
Could you add yourself in another commit?

Reply:
Added myself in the first commit, considering this resolved.